docs/documentation/platform/pam/overview.mdx

```diff
@@ -1,45 +1,68 @@
-
-### Infisical PAM Features
-
-#### Session Logging and Auditing
-
--   **Session Logging**: All user sessions are extensively logged, providing a detailed and searchable record of activities performed during a session.
--   **Audit Logging**: Every significant event, such as a user starting a session or accessing an account's credentials, is recorded in audit logs. This gives you complete visibility over your project.
-
-![Session Page](/images/pam/overview/session-page.png)
-
-#### Automated Credential Rotation
-
-Infisical PAM can automatically rotate account credentials to enhance your security posture.
-
-Here’s how it works:
-1.  **Add a Rotation Account**: On the resource level, you configure a "rotation account." This is a master or privileged account that has the necessary permissions to change the passwords of other accounts on that same resource.
-![Credential Rotation Account](/images/pam/overview/credential-rotation-account.png)
-
-2.  **Configure Rotation on Accounts**: For each individual account you want to rotate, you can simply enable rotation and set a desired interval (e.g., every 30 days).
-![Rotate Credentials Account](/images/pam/overview/rotate-credentials-account.png)
-
-Infisical will then use the rotation account on the resource to automatically update the credentials of the target account at the specified interval, eliminating credential staleness.
+## Core Concepts
+
+To successfully implement Infisical PAM, it is essential to understand the relationship between the following components:
+
+<CardGroup cols={3}>
+  <Card title="Gateway" icon="server">
+    A lightweight service deployed in your network that acts as a secure bridge to your private infrastructure.
+  </Card>
+  <Card title="Resource" icon="database">
+    The specific target you are protecting (e.g., a PostgreSQL database or an Ubuntu server) that resides behind a Gateway.
+  </Card>
+  <Card title="Account" icon="user-lock">
+    Specific credentials (username/password) used to authenticate against a Resource. One Resource can have multiple Accounts.
+  </Card>
+</CardGroup>
+
+### Relationship Model
+
+The hierarchy is structured as follows:
+
+1.  **Gateway**: Deployed once per network/VPC. It provides connectivity to all resources in that environment.
+2.  **Resource**: Configured within Infisical. It points to a specific IP/Host accessible by the Gateway.
+3.  **Account**: Defined under a Resource. Users request access to a specific *Account* on a *Resource*.
+
+## How it Works
+
+Infisical PAM uses a proxy-based architecture. When a user accesses a resource, their connection is routed securely through a Relay to your self-hosted Gateway, which then connects to the target resource. This ensures zero-trust access without exposing your infrastructure to the public internet.
+
+For a deep dive into the technical architecture and security model, see [Architecture](/documentation/platform/pam/architecture).
+
```

Comment:
it looks like this give an overview of the networking side of things but not really about the core essence of the PAM product